README-docker-compose.md

```diff
@@ -328,7 +328,7 @@ When using `docker-compose`, set `privileged: true` for individual containers in
 
 ```yml
 wb:
-  image: quay.io/centerforopenscience/waterbutler:develop
```

Comment:
`waterbutler:develop` is the wrong image and the correct one is `wb:develop`, see https://github.com/CenterForOpenScience/osf.io/commit/6b2232f1bc50fdd1c3009f5b65499e078ee80056.